pipeline/translate/splitter.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""
+Splits a dataset to chunks. Generates files in format file.00.zst, file.01.zst etc.
+
+Example:
+    python splitter.py \
+        --output_dir=test_data \
+        --compression_cmd=zstd \
+        --num_parts=10 \
+        --output_suffix=.ref \
+        test_data/corpus.en.zst
+"""
+
+import argparse
+import os
+import subprocess
+from contextlib import ExitStack
+
+
+def split_file(mono_path, output_dir, num_parts, compression_cmd, output_suffix=""):
+    os.makedirs(output_dir, exist_ok=True)
```

Comment:
suggestion (non-blocking): consider adding some logging to this function. Perhaps of the subprocess commands run, and maybe around the files created and what to expect in them (this can help debugging in the future if, eg: it seems like a file is missing, or has incomplete contents).

Reply:
added